### PR TITLE
OSOE-398: Update all NuGet packages to latest in UI-Testing-Toolbox

### PR DESCRIPTION
**Pull request by MZole**
[OSOE-398](https://lombiq.atlassian.net/browse/OSOE-398)

### CODE REVIEW

File: Lombiq.Tests.UI/Lombiq.Tests.UI.csproj [hunk 1]
```diff
@@ -51,22 +51,25 @@
...
+    <PackageReference Include="Atata.HtmlValidation" Version="2.2.0" />
+    <PackageReference Include="Atata.WebDriverExtras" Version="2.2.0" />
+    <PackageReference Include="Azure.Storage.Blobs" Version="12.14.0" />
     <PackageReference Include="Ben.Demystifier" Version="0.4.1" />
     <PackageReference Include="Codeuctivity.ImageSharpCompare" Version="2.0.76" />
     <PackageReference Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="6.0.1" />
     <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="6.0.0" />
     <PackageReference Include="Microsoft.SqlServer.DACFx" Version="160.6161.0" />
+    <!-- Can't update to Microsoft.SqlServer.SqlManagementObjects v170.8.0 or later until an Orchard upgrade:
+    Its Microsoft.Data.SqlClient dependency is higher than that required by OrchardCore.Data.YesSql -->
```
Comment (Piedone) [inline Lombiq.Tests.UI/Lombiq.Tests.UI.csproj]:
```suggestion
    Its Microsoft.Data.SqlClient dependency is higher than that required by OrchardCore.Data.YesSql. -->
```

File: Lombiq.Tests.UI/Lombiq.Tests.UI.csproj [hunk 2]
```diff
@@ -51,22 +51,25 @@
...
     <PackageReference Include="Microsoft.SqlServer.SqlManagementObjects" Version="161.47027.0" />
     <PackageReference Include="Mono.Posix.NETStandard" Version="5.20.1-preview" />
     <!-- During Orchard upgrades, this needs to be kept up to date with the version Orchard uses. -->
     <PackageReference Include="NLog.Web.AspNetCore" Version="4.14.0" />
     <PackageReference Include="Selenium.Axe" Version="4.0.2" />
-    <PackageReference Include="Shouldly" Version="4.0.3" />
+    <PackageReference Include="Shouldly" Version="4.1.0" />
+    <PackageReference Include="System.Drawing.Common" Version="5.0.0" />
```
Comment (Piedone) [inline Lombiq.Tests.UI/Lombiq.Tests.UI.csproj:72]: Is this necessary without touching DACFx?

**Review by Piedone — CHANGES_REQUESTED** [review]

File: Lombiq.Tests.UI/Lombiq.Tests.UI.csproj (hunk 2 above)
Comment (MZole) [reply Lombiq.Tests.UI/Lombiq.Tests.UI.csproj:72]: Yes, Shouldly v4.0.3 depends on System.Drawing.Common v5.0.0, but the newest Shouldly version is not depending on Drawing, thus our version of Drawing would be 4.7.0 (turns out the older DACFx and SqlManagementObjects only require this version), which is not compatible with our implementations in visual testing. But once the migration is done, this won't be necessary.

Comment (Piedone) [reply Lombiq.Tests.UI/Lombiq.Tests.UI.csproj:72]: I see.

**Review by Piedone — APPROVED** [review]

**Pull request merged**